modules/nw-ovn-kubernetes-checking-live-migration-metrics.adoc

```diff
@@ -26,21 +26,21 @@ Metrics are available to monitor the progress of the limited live migration. Met
 +
 [source,terminal]
 ----
-$ oc create token prometheus-k8s -n openshift-monitoring
+$ token=`oc create token prometheus-k8s -n openshift-monitoring`
 ----
 +
 .Example output
 +
 [source,terminal]
 ----
-eyJhbGciOiJSUzI1NiIsImtpZCI6IlZiSUtwclcwbEJ2VW9We...
+eyJhbGciOiJSUzI1NiIsImtpZCI6I6...
```

Comment:
the output should be removed

Reply:
thus `$ token=`oc create token prometheus-k8s -n openshift-monitoring` can be moved to before line 43.

Reply:
@zhaozhanqi Removed. Thanks!